src/Articulate.Web/App_Plugins/Articulate/Themes/VAPOR/Views/Post.cshtml

```diff
@@ -4,6 +4,30 @@
 @{
     Layout = "Master.cshtml";
     ViewBag.CssBodyClass = "post-template";
+    string imageCropUrl = null;
+
+    if (!Model.PostImageUrl.IsNullOrWhiteSpace())
+    {
+        imageCropUrl = Model.GetCropUrl("postImage", "blogPost");
+    }
+}
+
+@section socialMetaTags {
```

Comment:
I would prefer this to be a helper, the reason being is that i want to keep template creation/modification to be as simple as possible, I don't want everyone to have to be knowledgeable about exactly how all of this works